src/main/scala/tech/cryptonomic/conseil/config/Platforms.scala

```diff
@@ -52,7 +52,7 @@ object Platforms extends LazyLogging {
   }
 
   /** collects all config related to a tezos network */
-  final case class TezosConfiguration(network: String, depth: Depth = Newest, nodeConfig: TezosNodeConfiguration) extends PlatformConfiguration
+  final case class TezosConfiguration(network: String, depth: Depth = Newest, nodeConfig: TezosNodeConfiguration, headHash: Option[String] = None) extends PlatformConfiguration
```

Comment:
To be honest, I had the feeling that `depth` didn't belong here, and more-so now that we're adding `headHash`.
I'll explain myself: the Tezos configuration was meant to contain all values needed to define how to connect to the tezos nodes.
I would argue that `LorreConfiguration` is a best choice here, since it keeps all relevant values concerned with how Lorre is executed, independently of the platform it connects to.